ci-operator/step-registry/telcov10n/functional/ocp-cnf-tests/telcov10n-functional-ocp-cnf-tests-ref.yaml

```diff
@@ -0,0 +1,93 @@
+ref:
+  as: telcov10n-functional-ocp-cnf-tests
+  from_image:
+    namespace: ci
+    name: telco-bastion
+    tag: latest
+  commands: telcov10n-functional-ocp-cnf-tests-commands.sh
+  timeout: 4h
+  resources:
+    requests:
+      cpu: 100m
+      memory: 200Mi
+  credentials:
+  - namespace: test-credentials
+    name: kni-qe-41
+    mount_path: /var/run/ssh-key
+  - namespace: test-credentials
+    name: bastion-ip-addr
+    mount_path: /var/run/bastion-ip-addr
+  - namespace: test-credentials
+    name: jenkins-credentials
+    mount_path: /var/run/jenkins-credentials
+  env:
+  - name: LOCK_LBL_HOST
```

Comment:
Hi, 
when you provide a default value for an env var, it makes is not mandatory, meaning that a user can trigger it without providing a value. 
In most cases I know, this var (as well as some others) are mandatory.

Reply:
remove default